DEVELOPMENT.md

```diff
@@ -49,7 +50,19 @@ npm run bootstrap-react-app
 npm run start-react-app
 ```
 
-## Rebuild and Install Packages To the Generated App During Development
+### Frontend Libs 
+
+To update versions of @haulmont/jmix-rest and @cuba-platform/react in generated app [react-app-scr](react-app-scr)
```

Comment:
it's not necessary until major version bump in packages. I'd suggest to remove this